### ISSUE TITLE
OJS 3.4.0.8: menu in the top left corner does not work when accessed via smartphone

### ISSUE THREAD

**Issue by supriyadi354** [pkp/pkp-lib#10838]
Please provide a solution regarding the home menu (menu in the top left corner) which is not active when accessed using a mobile browser. If using a laptop, there are no problems.

![Image](https://github.com/user-attachments/assets/5c66fbbc-09bc-44b7-ab14-bc3977740ac5)
[https://drive.google.com/file/d/143apsqmFkjtXeEdI2g1F1dsWJ6pFXrhY/view?usp=sharing](url)
Warm regards,

**Comment by asmecher** [comment]
Cross-posted: https://forum.pkp.sfu.ca/t/ojs-3-4-0-8-menu-in-the-top-left-corner-does-not-work-when-accessed-via-smartphone/94676
